jib-maven-plugin/pom.xml

```diff
@@ -271,10 +271,9 @@
         </dependencies>
         <executions>
           <execution>
-            <id>default-cli</id>
             <phase>validate</phase>
             <goals>
-              <goal>format</goal>
+              <goal>check</goal>
```

Comment:
The `maven-checkstyle-plugin` below handles the google style check, so I believe we could just remove the entire `<executions>` block here.

Reply:
Fixed

Reply:
Looks like we actually should have fmt:check be run in validate as the checkstyle doesn't catch everything haha.